akka-cluster/src/test/scala/akka/cluster/VectorClockPerfSpec.scala

```diff
@@ -59,7 +59,7 @@ class VectorClockPerfSpec extends AnyWordSpec with Matchers {
     vc1.compareTo(vc2) should ===(order)
   }
 
-  def !==(vc1: VectorClock, vc2: VectorClock): Unit = {
+  def notEqual(vc1: VectorClock, vc2: VectorClock): Unit = {
```

Comment:
Manually fix: `multiarg infix syntax looks like a tuple and will be deprecated`,
take from [PR 28991](https://github.com/akka/akka/pull/28991/files#diff-05445b98f17ff01f5d4622290675a0ea)
MultiArgInfix scalafix rule should not cover this case (rename method)